src/docs/troubleshooting/key-bindings.md

```diff
@@ -15,6 +15,8 @@ If you don't want to [upgrade your terminal](terminal-recommendations), you can
 <!-- prettier-ignore -->
 - Comment line: use <Key>ctrl+_</Key> (underscore), not <Key>ctrl+/</Key>.
 - On a Mac: For all bindings, use <Key>^ Control</Key>, not <Key>⌘ Command</Key>.
+    - On MacOs >= 15.0.0 , the key binding <Key>^+enter</Key> is mapped to "Show contextual menu" by default. This interferes with the "Run Query" key binding of Harlequin. Instead of setting up alternative key bindings, you can disable this shortcut in MacOS under: 
+`System Settings -> Keyboard -> Keyboard Shortcuts... -> Keyboard -> turn "Show contextual menu" off`
```

Comment:
```suggestion
    - On MacOs >= 15.0.0 , the key binding <Key>^+enter</Key> is mapped to "Show contextual menu" by default. This interferes with the "Run Query" key binding of Harlequin. Instead of setting up alternative key bindings, you can disable this shortcut in MacOS by navigating to: System Settings -> Keyboard -> Keyboard Shortcuts... -> Keyboard -> turn "Show contextual menu" off.
```